src/crypto/sha256.cpp

```diff
@@ -492,6 +500,7 @@ std::string SHA256AutoDetect()
 {
     std::string ret = "standard";
 #if defined(USE_ASM) && (defined(__x86_64__) || defined(__amd64__) || defined(__i386__))
+    (void)AVXEnabled;
```

Comment:
nit: Presuming this line is to avoid an unused function warning, how about instead either:
* Commenting as to its purpose
* Defining it in an `#else` block below
* Only defining `AVXEnabled` when `defined(ENABLE_AVX2) && !defined(BUILD_BITCOIN_INTERNAL)`?

Reply:
I added a comment.

I'd rather not add logic around the compilation of the function itself, as it may grow increasingly complex in the future (if we also have avx1 code, for example). It's in an anonymous namespace anyway, so if it's unused, the compiler will drop it.